water-gui.py

```diff
@@ -132,34 +146,30 @@ def register_page(self) -> None:
         "prompt to enter new things."
         self.rpage = Frame(self.root)
         self.rpage.grid()
-        setSize(self.root, 505, 115)
+        setSize(self.root, 505, 90)
         # data variables to prompt
         self.reading = DoubleVar()
         self.menu_selection = IntVar()
         self.per_unit = ProductMap()
         self.product_index = self.per_unit.get_list()
         self.unit_count = DoubleVar()
-        # water amount
-        entry_l = Label(self.rpage, text="1. Introduzca la lectura del odometro", bg="whitesmoke", fg="black",
-        font=("Calibri", "12", "bold")).grid(row=0, column=0, sticky="ew")
-        entry_space = Entry(self.rpage, width=40, textvariable=self.reading).grid(row=0, column=1, sticky="ew")
         # client prompt
-        entry_c = Label(self.rpage, text="2. Producto vendido", bg="whitesmoke", fg="black",
-        font=("Calibri", "12", "bold")).grid(row=1, column=0, sticky="ew")
+        entry_c = Label(self.rpage, text="1. Producto vendido", bg="whitesmoke", fg="black",
+        font=("Calibri", "12", "bold")).grid(row=0, column=0, sticky="ew")
         self.client = get_menubutton(self.rpage,
                                      self.product_index, # use a list shared by all
                                      self.menu_selection,
-                                     row=1,
+                                     row=0,
                                      column=1)
         # unit prompt
-        entry_u = Label(self.rpage, text="3. Unidades vendidas", bg="whitesmoke", fg="black",
-        font=("Calibri", "12", "bold")).grid(row=3, column=0, sticky="ew")
-        units_space = Entry(self.rpage, width=40, textvariable=self.unit_count).grid(row=3, column=1, sticky="ew")
+        entry_u = Label(self.rpage, text="2. Unidades vendidas", bg="whitesmoke", fg="black",
+        font=("Calibri", "12", "bold")).grid(row=1, column=0, sticky="ew")
+        units_space = Entry(self.rpage, width=40, textvariable=self.unit_count).grid(row=1, column=1, sticky="ew")
         # "check" and "cancel" buttons
-        check = Button(self.rpage, text="Entregar ahora", command=self.evaluate, font=("Calibri", "14", "bold"), fg="whitesmoke",
-        bg="green").grid(row=4, column=0, sticky="ew")
-        cancel = Button(self.rpage, text="Cancelar", command=self.reg_cancel, font=("Calibri", "14", "bold"), fg="whitesmoke",
-        bg="red").grid(row=4, column=1, sticky="ew")
+        check = Button(self.rpage, text="Entregar ahora", command=self.evaluate, font=("Calibri", "12", "bold"), fg="whitesmoke",

```

Comment:
I am modifying the widget sizes tentatively, I'm still not sure of those values.